doc/api/tls.md

```diff
@@ -876,6 +876,30 @@ socket.on('end', () => {
 });
 ```
 
+Passing in `net.Socket` will start a new instance.
+To upgrade an existing instance of `net.Socket` to a 
+`tls.TLSSocket`, pass it to `tls.connect()` as 
+the socket option: 
+
+```js
+const { Socket } = require('net');
+const tls = require('tls');
+const sock = new Socket();
```

Comment:
Yes, you're right! Just updated to reflect that variable and changed the `irc.freenode.net` to `https://example.org:443/`.